### PR TITLE
Minimal Python Bindings for ApproxMC

### PR DESCRIPTION
**Pull request by Eric-Vin**
First off, thank you so much for the great tool! I'm using ApproxMC in some experiments for a paper I'm writing so I wrote some minimal Python bindings for it based off of the CryptoMiniSat ones so that I could better integrate it. I tried to follow the same style and naming conventions as well. Unfortunately, I couldn't figure out how to integrate it with your build system, but when built locally it passes a few basic tests I wrote for it.

### CODE REVIEW

File: src/approxmc.cpp [hunk 1]
```diff
@@ -199,6 +199,11 @@ DLL_PUBLIC bool AppMC::get_reuse_models()
     return data->conf.reuse_models;
 }
 
+DLL_PUBLIC void AppMC::setup_vars()
```
Comment (msoos) [inline src/approxmc.cpp:202]: You declare and define this function but you don't use it, right? Can we remove it?

**Review by msoos — CHANGES_REQUESTED** [review]
First of all, amazing work, thank you so much!! I really-really like it. I think I only found one issue with `setup_vars` that you seem to declare&define but not use? If that's correct, can we remove it? Or maybe I'm wrong, please let me know!

Thanks a lot, amazing work,

Mate

File: src/approxmc.cpp (hunk 1 above)
Comment (Eric-Vin) [reply src/approxmc.cpp:202]: Glad to hear you like the code! It's my first time writing Python bindings or any decent chunk of C++ so definitely let me know if anything looks funky. This functions was for the Unigen bindings I wrote (https://github.com/meelgroup/unigen/pull/7) to setup the internal solver for Unigen without having to count everything, but please let me know if there's a better way to do that! I'll go into more detail on that PR.

Comment (msoos) [reply src/approxmc.cpp:202]: Ahha, I see! Merging now :)

**Comment by Eric-Vin** [comment]
Woops, I realized that I left a memory leak open and added a fix for that, but I forgot to add a semicolon after it so these won't actually build. It's on line 65, and once it's adding everything works as intended. Sorry about that!

**Comment by msoos** [comment]
I fixed it, pushed it. Unfortunately, the system is also lacking a simple `README` under `python/`, and the main `README.md` is also lacking a mention about the python bindings -- a shame because you put so much effort into it, let's advertise it! :)

**Pull request merged**